src/nn/models.py

```diff
@@ -210,6 +212,27 @@ def regularization(self):
     def calculate_loss(self, y_hat, y):
         return mse_loss(y_hat.squeeze(1), y)
 
+    def _pred_metrics(self, prefix: str, y_hat: torch.Tensor, y: torch.Tensor) -> RegLoaderMetrics:
```

Comment:
I think we should stop hardcoding r2 (and mse) as metric since we want to use this code for binary/discrete phenotypes as well. I understand though that we need to change that all over our code, so changing in this one particular place is not that important but in general, I guess, the new code should be more flexible.